### DISCUSSION TITLE
Нас поддерживает RUVDS!

### DISCUSSION THREAD

**Discussion by jarvis394** [jarvis394/geekr#147] [Show and tell]
### Новый сервер

Теперь на сервере я смогу сделать кластер из моего бекенда, могу навесить туда аналитики, могу делать всё что угодно! Для пользователей **geekr.** новый сервер означает более быструю загрузку контента, если они авторизовались (для всех остальных ничего не поменялось).

**Comment by centralhardware** [comment]
То есть, если авторизовались?

**Reply by jarvis394** [reply]
То есть вошли в аккаунт.
